index.php

```diff
@@ -21,7 +21,12 @@
 	// Load posts loop.
 	while ( have_posts() ) {
 		the_post();
-		get_template_part( 'template-parts/content/content' );
+
+		if ( 'excerpt' === get_theme_mod( 'display_excerpt_or_fullpost', 'excerpt' ) ) {
+			get_template_part( 'template-parts/content/content-excerpt' );
+		} else {
+			get_template_part( 'template-parts/content/content' );
+		}
```

Comment:
Hmmm thinking about this one some more, would it perhaps be better written like this instead?

```
get_template_part(
	'template-parts/content/content-excerpt',
	'excerpt' === get_theme_mod( 'display_excerpt_or_fullpost', 'excerpt' ) ? 'excerpt : ''
);
```

Reply:
Scratch that, what you have there is more readable :+1: